keras_cv/layers/preprocessing/random_zoom_test.py

```diff
@@ -0,0 +1,152 @@
+# Copyright 2022 The KerasCV Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import numpy as np
+import tensorflow as tf
+from absl.testing import parameterized
+
+from keras_cv.layers.preprocessing.random_zoom import RandomZoom
+
+
+class RandomZoomTest(tf.test.TestCase, parameterized.TestCase):
+    def _run_test(self, height_factor, width_factor):
```

Comment:
Same as last PR -- let's rename this test_output_shape and parameterize it directly instead of calling it from other parameterized tests

Reply:
done